src/draco/tests/test_data_utils.py

```diff
@@ -10,7 +10,7 @@
 draco: Draco = Draco()
 learn_data: dict[str, dict[str, str]] = {}
 
-root_path: Path = Path(__file__).resolve().parents[2]
+root_path: Path = Path(__file__).resolve().parents[3]
```

Comment:
We moved the module sources under `src/` so we must look one parent ahead.

Reply:
What's the point of moving the source? There is only one folder so it seems unnecessary.

Reply:
We will still need some minimal tooling around Pyodide and JupyterLite. I planned to move those files under `src/`.
Do you suggest to keep it top-level.

Reply:
Let me see the end result.

Reply:
I ended up dropping `src/`, as I compressed the JupyterLite-related tooling into a single file (`docs/jupyterlite_nb_patch.py`)